client/app/styles/index.styl

```diff
@@ -179,3 +293,56 @@ aside
                 height      6em
                 width       100%
                 margin      2em 0em
+
+// ONBOARDING NOTIFICATION
```

Comment:
This comment should be updated for this app, there is no relation to the cozy-proxy onboarding here.